<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessBoard</class>
 <widget class="QMainWindow" name="ChessBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>900</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>249</red>
        <green>249</green>
        <blue>249</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>68</red>
        <green>72</green>
        <blue>81</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>249</red>
        <green>249</green>
        <blue>249</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>68</red>
        <green>72</green>
        <blue>81</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>68</red>
        <green>72</green>
        <blue>81</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>68</red>
        <green>72</green>
        <blue>81</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Chess</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Game"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuE_dit">
    <property name="title">
     <string>E&amp;dit</string>
    </property>
    <widget class="QMenu" name="menu_Theme">
     <property name="title">
      <string>&amp;Theme</string>
     </property>
     <addaction name="actionSlate"/>
     <addaction name="actionClassic"/>
     <addaction name="actionLight"/>
     <addaction name="actionBlue"/>
     <addaction name="actionCustom"/>
    </widget>
    <addaction name="menu_Theme"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Rules"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuE_dit"/>
   <addaction name="menu_About"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Rules">
   <property name="text">
    <string>&amp;Rules</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_New_Game">
   <property name="text">
    <string>&amp;New Game</string>
   </property>
  </action>
  <action name="actionSlate">
   <property name="text">
    <string>Slate</string>
   </property>
  </action>
  <action name="actionClassic">
   <property name="text">
    <string>Classic</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom...</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
